pkg/apis/flowcontrol/validation/validation.go

```diff
@@ -332,6 +342,42 @@ func ValidatePriorityLevelConfigurationCondition(condition *flowcontrol.Priority
 	return allErrs
 }
 
+// ValidateNonResourceURLPath validates non-resource-url path by following rules:
+//   1. Slash must be the leading character of the path
+//   2. A non-resource-url path must not end up with slash
```

Comment:
why?

Reply:
[the api convention doc](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api-conventions.md#verbs-on-resources) doesnt explicitly say there cant be ending slash in path. but based on the observation of all the existing api paths in kube-apiserver, it's probably fine to tighten that usage?

Reply:
convinced, the path validation should respect all the existing paths served by apiserver